app/views/planning_applications/confirm_validation.html.erb

```diff
@@ -23,9 +23,13 @@
       <div class="govuk-hint" id="planning-application-validated-at-hint">
         Will be marked as valid from <%= @planning_application.valid_from_date.to_date.to_fs %>
       </div>
-      <%= form.govuk_radio_buttons_fieldset :make_public, legend: {text: "Publish application on BOPS Public Portal?"} do %>
-        <%= form.govuk_radio_button :make_public, true, label: {text: "Yes"} %>
-        <%= form.govuk_radio_button :make_public, false, label: {text: "No"} %>
+      <% if @planning_application.pre_application? %>
```

Comment:
Might make sense for this conditional to be the same as the one in the other file.